packages/wallets/src/metamask/pages/walletOperations.page.ts

```diff
@@ -22,17 +22,17 @@ export class WalletOperationPage {
     this.page = page;
     this.nextButton = this.page.getByTestId('page-container-footer-next');
     this.confirmButton = this.page.getByTestId('confirm-footer-button');
-    this.rejectButton = this.page.getByTestId('page-container-footer-cancel');
+    this.approvalRejectButton = this.page.getByTestId(
```

Comment:
Does "reject" wording applies only for approval modal?

Reply:
renamed to "cancel"